helloworld.css

```diff
@@ -0,0 +1,4 @@
+.hdr {
```

Comment:
You could have used h1 tag instead of making a new class to modify the h1 element